File: Sem4/MPP/cs/lab2/Domain/Bilet.cs
namespace Domain;

public class Bilet:Entity<long>
{
    public Bilet(string buyer, int seatsNumber, Spectacol spectacol)
    {
        Buyer = buyer;
        SeatsNumber = seatsNumber;
        SpectacolC = spectacol;
    }
    
    public string Buyer { get; set; }   
    public int SeatsNumber { get; set; }
    public Spectacol SpectacolC { get; set; }
    
    
    
}

File: Sem4/MPP/java/lab2/Domain/src/main/java/ubb/scs/mpp/Bilet.java
package ubb.scs.mpp;

public class Bilet extends Entity<Long> {
    String buyer;
    Integer seats_number;
    Spectacol spectacol;

    public String getBuyer() {
        return buyer;
    }

    public void setBuyer(String buyer) {
        this.buyer = buyer;
    }

    public Integer getSeats_number() {
        return seats_number;
    }

    public void setSeats_number(Integer seats_number) {
        this.seats_number = seats_number;
    }

    public Spectacol getSpectacol() {
        return spectacol;
    }

    public void setSpectacol(Spectacol spectacol) {
        this.spectacol = spectacol;
    }

    public Bilet(String buyer, Integer seats_number, Spectacol spectacol) {
        this.buyer = buyer;
        this.seats_number = seats_number;
        this.spectacol = spectacol;
    }
}
